src/components/NcAppNavigationNewItem/NcAppNavigationNewItem.vue

```diff
@@ -69,26 +69,26 @@
 		}"
 		class="app-navigation-entry">
 		<!-- New Item -->
-		<div class="app-navigation-entry-div" @click="handleNewItem">
-			<div :class="{ [icon]: !loading }"
+		<button class="app-navigation-entry-div" @click="handleNewItem">
+			<span :class="{ [icon]: !loading }"
 				class="app-navigation-entry-icon">
 				<NcLoadingIcon v-if="loading" />
 				<slot v-else name="icon" />
-			</div>
+			</span>
 
-			<span v-if="!newItemActive" class="app-navigation-new-item__name" :title="name">
+			<button v-if="!newItemActive" class="app-navigation-new-item__name" :title="name">
```

Comment:
I think this second button within a button is wrong

Reply:
Thanks @susnux .... I mixed this up working between `stable7` and `master`